renderer/components/traffic-lights.js

```diff
@@ -0,0 +1,117 @@
+
+import {remote} from 'electron';
+import React from 'react';
+
+export default () => (
+  <div className="root">
+    <div className="traffic-lights">
+      <button type="button" className="traffic-light traffic-light-close" onClick={() => remote.app.quit()} id="close"/>
+      <button type="button" className="traffic-light traffic-light-minimize" onClick={() => remote.BrowserWindow.getFocusedWindow().minimize()} id="minimize"/>
+      <button type="button" disabled className="traffic-light traffic-light-maximize" onClick={() => false && remote.BrowserWindow.getFocusedWindow().maximize()} id="maximize"/>
+    </div>
+    <style jsx>{`
+  .root {
+    position: absolute;
+    top: 0;
+    left: 16px;
+  }
+  .traffic-lights {
+    top: 1px;
+    z-index: 100;
+    left: 8px;
+    -webkit-app-region: no-drag;
+  }
+  .traffic-lights > .traffic-light-close, .traffic-lights:hover > .traffic-light-close, .traffic-lights:active > .traffic-light-close {
+    background-color: #ff6159;
+  }
+  .traffic-lights > .traffic-light-close:active:hover, .traffic-lights:hover > .traffic-light-close:active:hover, .traffic-lights:active > .traffic-light-close:active:hover {
+    background-color: #bf4942;
+  }
+  .traffic-lights > .traffic-light-minimize, .traffic-lights:hover > .traffic-light-minimize, .traffic-lights:active > .traffic-light-minimize {
+    background-color: #FFBF2F;
+  }
+  .traffic-lights > .traffic-light-minimize:active:hover, .traffic-lights:hover > .traffic-light-minimize:active:hover, .traffic-lights:active > .traffic-light-minimize:active:hover {
+    background-color: #995700;
+  }
+  .traffic-lights > .traffic-light-maximize, .traffic-lights:hover > .traffic-light-maximize, .traffic-lights:active > .traffic-light-maximize {
+    // background-color: #28c941;
+  }
+  .traffic-lights > .traffic-light-maximize:active:hover, .traffic-lights:hover > .traffic-light-maximize:active:hover, .traffic-lights:active > .traffic-light-maximize:active:hover {
+    // background-color: #1d9730;
```

Comment:
Why commented out? Should rather remove it then.

Reply:
I copied it from codepen with the green button for fullscreen there, didn't know if we wanted it and didn't want to remove the css for it. Do you think I should remove it completely or prepare the component with a prop to enable the fullscreen option?

Reply:
Either of those should happen. Up to @skllcrn whether we should keep it or not.

Reply:
Since fullscreen is an option in the player itself, it makes sense to enable it here too?